config.py

```diff
@@ -1348,6 +1350,11 @@ def get_t1_from_meeg(bids_path):
                "instead by setting spatial_filter='ica'.")
         raise ValueError(msg)
 
+if conditions is None:
+    msg = (f'Please indicate the name of your conditions in your configuration. '
+           f'Currently the `conditions` parameter is empty.')
```

Comment:
We don't need the `f`s here, as we're not using any variables in the string :)

```suggestion
    msg = ('Please indicate the name of your conditions in your configuration. '
           'Currently the `conditions` parameter is empty.')
```

Reply:
Ah! I see, the "f" is when you have a parameter to evaluate, I copied it from other msg :) Thanks for fixing it!

Reply:
Yes so the `f` indicates an "f-string":

https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings